src/main/java/com/Saurabh/GitHubAiAPI/StringOperations.java

```diff
@@ -43,7 +43,7 @@ public static void main(String[] args) {
 
         // Validate input
         if (input == null || input.isEmpty()) {
-            System.out.println("Invalid input. Please enter a non-empty string.");
+            System.out.println("Invald input. Please enter non-empty string.");
```

Comment:
There are multiple typos in the print statements. Please correct them to 'Number of vowels:', 'String with replaced characters:', and 'Concatenated string:'